ui-participant/src/Navbar.tsx

```diff
@@ -68,7 +68,7 @@ export default function Navbar(props: NavbarProps) {
       <button
         aria-controls={dropdownId} aria-expanded="false" aria-label="Toggle navigation"
         className="navbar-toggler"
-        data-bs-toggle="collapse" data-bs-target={`#${CSS.escape(dropdownId)}`}
+        data-bs-toggle="collapse" data-bs-target={`#${dropdownId}`}
```

Comment:
My guess is that the newer version of Bootstrap handles the escaping now, but I haven't found proof. I am wondering if there are other parts of the code that would need to be updated, if true (I haven't checked yet)

Reply:
It's the opposite -- I updated us to a later version and it broke things.  React `useId` deliberately includes weird characters that then throw off bootstrap.  So in addition to getting rid of the escaping, we should switch line 60 above to use lodash unique id instead `_uniqueId('navDropdown')`